--- spice-xpi-2.6/configure.ac.orig	2011-08-25 11:07:12.000000000 +0200
+++ spice-xpi-2.6/configure.ac	2011-09-23 16:21:42.335742663 +0200
@@ -96,14 +96,14 @@
    )
 AC_SUBST(LIBRT)
 
-SPICE_NONPKGCONFIG_LIBS+=" $LIBM $LIBRT"
+SPICE_NONPKGCONFIG_LIBS="$SPICE_NONPKGCONFIG_LIBS $LIBM $LIBRT"
 
 SPICE_REQUIRES=""
 
 PKG_CHECK_MODULES(LOG4CPP, log4cpp)
 AC_SUBST(LOG4CPP_CFLAGS)
 AC_SUBST(LOG4CPP_LIBS)
-SPICE_REQUIRES+=" log4cpp"
+SPICE_REQUIRES="$SPICE_REQUIRES log4cpp"
 
 # Add parameter for (partial) static linkage of spice client.
 # this is used to achive single binary package for all (?) distros.
@@ -117,7 +117,7 @@
 PKG_CHECK_MODULES(XUL, libxul-embedding >= 1.9 nspr >= 4.7.1)
 AC_SUBST(XUL_CFLAGS)
 AC_SUBST(XUL_LIBS)
-SPICE_REQUIRES+=" libxul-embedding >= 1.9 nspr >= 4.7.1"
+SPICE_REQUIRES="$SPICE_REQUIRES libxul-embedding >= 1.9 nspr >= 4.7.1"
 
 # Find xpidl
 for i in `pkg-config --variable=libdir libxul`/xpidl ; do
