--- spice-xpi-2.5/configure.ac.orig	2011-06-07 10:36:54.000000000 +0200
+++ spice-xpi-2.5/configure.ac	2011-07-20 11:04:03.150855074 +0200
@@ -96,19 +96,19 @@
    )
 AC_SUBST(LIBRT)
 
-SPICE_NONPKGCONFIG_LIBS+=" $LIBM $LIBRT"
+SPICE_NONPKGCONFIG_LIBS="$SPICE_NONPKGCONFIG_LIBS $LIBM $LIBRT"
 
 SPICE_REQUIRES=""
 
 PKG_CHECK_MODULES(LOG4CPP, log4cpp)
 AC_SUBST(LOG4CPP_CFLAGS)
 AC_SUBST(LOG4CPP_LIBS)
-SPICE_REQUIRES+=" log4cpp"
+SPICE_REQUIRES="$SPICE_REQUIRES log4cpp"
 
 PKG_CHECK_MODULES(X11, x11)
 AC_SUBST(X11_CFLAGS)
 AC_SUBST(X11_LIBS)
-SPICE_REQUIRES+=" x11"
+SPICE_REQUIRES="$SPICE_REQUIRES x11"
 
 # Add parameter for (partial) static linkage of spice client.
 # this is used to achive single binary package for all (?) distros.
@@ -122,7 +122,7 @@
 PKG_CHECK_MODULES(XUL, libxul-embedding >= 1.9 nspr >= 4.7.1)
 AC_SUBST(XUL_CFLAGS)
 AC_SUBST(XUL_LIBS)
-SPICE_REQUIRES+=" libxul-embedding >= 1.9 nspr >= 4.7.1"
+SPICE_REQUIRES="$SPICE_REQUIRES libxul-embedding >= 1.9 nspr >= 4.7.1"
 
 # Find xpidl
 for i in `pkg-config --variable=libdir libxul`/xpidl ; do
